### DISCUSSION TITLE
Should we release 0.70.11?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#71] [Patches]
Released, please refer to #77
---
Current Release: [0.70.10](https://github.com/facebook/react-native/releases/tag/v0.70.10)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. bump to do:
   * Metro 0.72.4
   * CLI 9.3.3 [react-native-community/cli@9.x (commits)](https://github.com/react-native-community/cli/commits/9.x)
1. https://github.com/facebook/react-native/commit/cd56347dca4e948f5038643bcd804c41f037727a

#### Local commits to backport to main

1.

**Comment by kelset** [comment]
from @tido64:
> Just found that Xcode defaults to 'Release' when running react-native run-ios under specific setups. Fix here: https://github.com/react-native-community/cli/pull/1957. Would be great if we could get this into 0.71.

**Reply by kelset** [reply]
this is the 0.70 release, hiding

**Comment by kelset** [comment]
from @robhogan:
> Nominate this one for the RN-side fix to https://github.com/facebook/react-native/issues/36794 (to accompany a bump to Metro 0.72.4).
> - https://github.com/facebook/react-native/pull/37501 (https://github.com/facebook/react-native/commit/671ea383fe45dd9834a0c0481360de050df7f0c9)

**Reply by kelset** [reply]
bump to do:
* Metro 0.72.4
* CLI 9.3.3 https://github.com/react-native-community/cli/commits/9.x

**Comment by nikit-psl** [comment]
I'm not sure if it's the right place to ask this, is it possible to release the hotfix [Crashing on Oppo/OnePlus Device](https://github.com/facebook/react-native/commit/c05d822f7daa92e8af2ec2cd97a9897425624cc2) for previous React Native versions as well? I saw that it was released for 0.70.10 but for us(version 0.67.5) and couple of other people, we are facing this for older react native versions as well and there are a lot of crashes daily because of it.

**Reply by kelset** [reply]
we only do new patch releases for versions within the [support window](https://github.com/reactwg/react-native-releases#which-versions-are-currently-supported)

**Reply by cipolleschi** [reply]
@nikit-psl thanks for the question, but, as @kelset said, we are not supporting version 0.67.5 anymore.
We strongly suggest you and your team to allocate some time to upgrade to a newer version. We are putting a lot of effort in trying to make every release less disruptive from the update perspective so that the teams can consistently keep their apps updated.

**Comment by cipolleschi** [comment]
We need this also for Xcode 15
- https://github.com/facebook/react-native/pull/37758

**Reply by kelset** [reply]
it's already in 0.70.10 https://github.com/reactwg/react-native-releases/discussions/66#discussioncomment-6120064

**Comment by harshad-kathiriya** [comment]
Hey, is this https://github.com/facebook/react-native/commit/c05d822f7daa92e8af2ec2cd97a9897425624cc2 added in v0.70.5? if not, could you please add that?

**Reply by kelset** [reply]
@harshad-kathiriya it's already in 0.70.10 -> https://github.com/facebook/react-native/releases/tag/v0.70.10

**Comment by joe-sam** [comment]
Prevent LogBox from crashing on long messages #38005 
https://github.com/facebook/react-native/pull/38005

**Reply by kelset** [reply]
tried cherry-picking, but the merge conflicts are fairly extensive. Please try to setup a PR directly against 0.70 branch so that we can merge in the next patch.
